src/server/util/spill_file.cc

```diff
@@ -0,0 +1,117 @@
+/** Copyright 2020-2021 Alibaba Group Holding Limited.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+#include "server/util/spill_file.h"
+#include <glog/logging.h>
+
+#include <memory>
+#include <string>
+#include "common/memory/payload.h"
+#include "common/util/base64.h"
+#include "common/util/status.h"
+#include "common/util/uuid.h"
+#include "io/io/local_io_adaptor.h"
+#include "server/memory/memory.h"
+#include "server/util/file_io_adaptor.h"
+
+namespace util {
+using vineyard::BulkStore;
+using vineyard::Payload;
+using vineyard::Status;
+void PutFixed32(std::string* dst, uint32_t value) {
```

Comment:
Is this an unused function?

Reply:
yes, maybe we could keep it for later use...?